app/mixins/async_tasks.js

```diff
@@ -0,0 +1,61 @@
+import Ember from "ember";
+const { getOwner } = Ember;
+
+export default Ember.Mixin.create({
```

Comment:
Excellent !! work here. Do we need to use this mixin with other controllers to avoid use of spinners in future ?

Reply:
You don't "need" to, but it's there as a helper to make the controllers leaner and smaller

Reply:
I don't want to enforce a practice, so I try it out, and see if it sticks : )
But I definitely don't like the way it's currently done (copy pasted every time)